portal/portal-impl/impl/src/java/org/sakaiproject/portal/charon/handlers/SiteHandler.java

```
@@ -850,6 +850,24 @@ public void includeTabs(PortalRenderContext rcontext, HttpServletRequest req,
 			rcontext.put("viewAsStudentLink", Boolean.valueOf(roleswapcheck)); // this will tell our UI if we want the link for swapping roles to display
 			rcontext.put("roleSwitchState", roleswitchstate); // this will tell our UI if we are in a role swapped state or not
 
+			int tabDisplayLabel = 1;
+			
+			if (loggedIn) 
+			{
+				Preferences prefs = PreferencesService.getPreferences(session.getUserId());
+				ResourceProperties props = prefs.getProperties("sakai:portal:sitenav");
+				try 
+				{
+					tabDisplayLabel = (int) props.getLongProperty("tab:label");
+				} 
+				catch (Exception any) 
+				{
+					
```

Lets not leave an empty catch statement, I would set it to some default here.